saved_data/authors.json

```diff
@@ -0,0 +1 @@
+[{"first_name":"Jane","last_name":"Grey"},{"first_name":"Charlse","last_name":"Dickens"},{"first_name":"Trust","last_name":"Simeon"}]
```

Comment:
### [optional]

Consider using `pretty_generate` To get a pretty output of JSON from an active record object.